package.json

```diff
@@ -13,7 +13,7 @@
     "clean": "rimraf --glob **/.next **/.turbo **/dist **/node_modules",
     "code:check": "pnpm install --frozen-lockfile && biome ci . & pnpm run sort-manifests --check & cspell --dot .",
     "code:fix": "biome check --write . & pnpm run sort-manifests",
-    "dev": "turbo run dev",
+    "dev": "turbo run dev --concurrency=20",
```

Comment:
Turbo's default concurrency is 10. In `dev` mode, there's now 11 tasks, so I'm bumping this up.